### PR TITLE
update doc/sql-databases.md

### PR DESCRIPTION
**Pull request by jfeltesse-mdsol**
- fix broken link
- linkify referenced files
- enable syntax highlighting of scala code
- `new DB()` :arrow_right: `DB()`, as in the code itself there's no `new`.

### CODE REVIEW

File: doc/sql-databases.md [hunk 1]
```diff
@@ -18,24 +18,28 @@ on-demand.
...
+"production", you might write something like this:
 
-    val db = new DB(new DBConfig("mysql", new DBParams("production", "127.0.0.1")))
+```scala
+  val db = DB(new DBConfig("mysql", new DBParams("production", "127.0.0.1")))
+```
 
 The connection parameters you can specify are defined in
-[`DBConfig.scala`](https://github.com/openzipkin/zipkin/blob/master/zipkin-anormdb/src/main/scala/com/openzipkin/zipkin/util/DBConfig.scala)
+[`DBConfig.scala`](https://github.com/openzipkin/zipkin/blob/master/zipkin-anormdb/src/main/scala/com/twitter/zipkin/util/DBConfig.scala)
```
Comment (codefromthecrypt) [inline doc/sql-databases.md:38]: was this change intentional?

**Comment by codefromthecrypt** [comment]
thanks for the help. only one nit

File: doc/sql-databases.md (hunk 1 above)
Comment (jfeltesse-mdsol) [inline doc/sql-databases.md:38]: yes, that's the broken link. 

`.../openzipkin/...` → 404

Comment (codefromthecrypt) [inline doc/sql-databases.md:38]: nice catch!
